azext_iot/central/models/devicepreview.py

```diff
@@ -7,10 +7,9 @@
 from azext_iot.central.models.enum import DeviceStatus
 
 
-class Device:
+class DevicePreview:
     def __init__(self, device: dict):
         self.approved = device.get("approved")
-        self.description = device.get("description")
```

Comment:
There is no description in `deviceV1` as well...is that intended?

Reply:
description does not exist for preview or v1 - removed  in both.